src/constants/index.js

```diff
@@ -13,7 +13,7 @@ export const LATEST = 'latest'
 export const RANDOM = 'random'
 
 export const SOCIALS = {
-  Email: 'kylie@kyliestewart.tech',
+  Email: 'mailto:kylie@kyliestewart.tech',
```

Comment:
I don't want to store this extra browser stuff in my constants, so I'm going to prefer a route that just modifies the broken button in place. Going to make that small change to this PR and merge it in. Thank you for opening :-)

Reply:
it does make more sense your way